src/common/utils/session-id-generator.ts

```diff
@@ -0,0 +1,28 @@
+import { Utils } from '@playkit-js/playkit-js';
+
+class SessionIdGenerator {
+  private static next: string = '';
+
+  private static init() {
+    SessionIdGenerator.next = `${Utils.Generator.guid()}:${Utils.Generator.guid()}`;
+  }
+
+  public static get() {
```

Comment:
this function is a getter but it creating a side effect of both init and setting next item.
Consider rename